src/strtree.h

```diff
@@ -14,18 +14,18 @@ typedef struct
 } npy_intp_vec;
 
 
-/* A resizable vector with pointers */
+/* A resizable vector with pointers to pygeos GeometryObjects */
 typedef struct
 {
     size_t n, m;
     GeometryObject **a;
-} geom_obj_vec;
+} pg_geom_obj_vec;
```

Comment:
@caspervdw suggestion: we should consider using "pg_" prefix on things that refer to pygeos GeometryObjects to keep them distinct from GEOSGeometry objects - that way it is a bit easier just in looking at the variable name to know which we're dealing with.

I went ahead and made that change here, but wanted to flag it specifically.